tpl/internal/go_templates/htmltemplate/hugo_template.go

```diff
@@ -17,6 +17,10 @@ import (
 	template "github.com/gohugoio/hugo/tpl/internal/go_templates/texttemplate"
 )
 
+// See https://github.com/golang/go/issues/59234
+// Moved here to avoid dependency on Go's internal/debug package.
+var debugAllowActionJSTmpl = false
```

Comment:
Ideal: If the choice is between backwards compatibility and security, then security wins.

Practical: It depends on how many sites will break, and to some extent, which sites. 

I will need to work with this PR to understand the effects before I can provide meaningful feedback and examples.

Reply:
I think it's worth it to have this default secure, but add an option for it in Hugo's security config. I don't think either of us can know how much this will break until it ... breaks.

Reply:
I'm fine with that approach.

Reply:
OK, the above seem to be restricted to just script inside back ticks, which should be pretty rare:

```js
var a = `{{.Title }}`;
```

Reply:
The only example usage of JS template literals that I can remember seeing is <https://github.com/gohugoio/hugo/issues/10707>.